monitoring/uss_qualifier/reports/tested_requirements.py

```diff
@@ -147,13 +171,33 @@ class TestedBreakdown(ImplicitDict):
     packages: List[TestedPackage]
 
 
-def generate_tested_requirements(report: TestRunReport, output_path: str) -> None:
+def generate_tested_requirements(
+    report: TestRunReport, config: TestedRequirementsConfiguration
+) -> None:
+    req_collections: Dict[
+        TestedRequirementsCollectionIdentifier, Set[RequirementID]
+    ] = {}
+    if "requirement_collections" in config and config.requirement_collections:
```

Comment:
Nit: is this equivalent?
```suggestion
    if config.has_field_with_value("requirement_collections"):
```

Reply:
It is equivalent and would be fine.  I generally take the first approach because the line will be found if I search for all instances of the `requirement_collections` field using my IDE whereas the second approach will not find the line.

Reply:
Indeed, makes sense :)